### ISSUE TITLE
[NEW BUILD - DEFAULT / 19w40b] Combat, System balance

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Skywanderers#1462]
So this is an important update, with some fondation for combat and proper system balance.
For now a lot of things are wip, but the goal will be to build and try out various ship design, balance out system / weapon value, and address / improve any shortcoming in the current combat experience.
Dont hesitate to send ship and give back feedback on your experience!
An update on weapons as it is now rather limited.

Please check the vid on how to spawn and AI and link a turret (or just do F5, and link a target computer to your turret to get AI turret ;))
https://youtu.be/cms1TzQbYJw

Thanks for testing the game as always!

**Comment by ZachZent** [comment]
Though this is the first iteration, the UI is very unfriendly for the user. I know you are looking for changes so here is my little mock-up. This is edited from the game Star Trek Online which has a UI system I always enjoyed
https://cdn.discordapp.com/attachments/416013956331601930/632148515040133120/unknown.png
For the status bars if you click on one of the tabs, you will get more details. Say for shield tab you see the recharge rate, shield health.
The center of the circle can be a spinning model of the ship like the core/drone. Even better would be a hologram with damaged blocks in red
It would obviously need a bit of changing to be fully Skywanderer style and not STO, but it is still an improvement

**Comment by tsunamayo** [comment]
Cool thanks for the screenshot! Yes it is very wip for now, at the mininum I will add some icon and likely some numbers tbd.
How do you click on a tab, your mouse is looked to the view or the ship control?
For the 3d preview what is the use, to get a view of where there is damage?

But maybe before jumping into replacement I would like to understand what is wrong/missing first. Like do you need to check your recharge rate in combat in case some of your shield is destroyed and you want to see where you stand?
Thanks

**Comment by ZachZent** [comment]
Oh yea, I forgot that some people use the mouse to steer. I changed mine a long time ago to wasd allowing me to look and fly. Even press tab turning off the HUD giving even more range to look. The different tabs was a idea for the future. Different NPC stations dedicated to each of the systems where the pilot can look.

The 3d view can be used for damage. I would love to see red damaged indicators for blocks damaged or destroyed. Would be even better to see a hologram view like in the old game versions before the feature was removed. You were able to see inside the ship including moving parts.

Truthfully the essentials are there minus clarity. You could leave it at that and it would all be fine. Different looks like the one I posted or any number of video game ship HUDs is mostly for style and to allow for non-essential information to be displayed. Four bars across the bottom left of the screen isn't exactly appealing nor does it really add any the potential for optional information where as an enhanced one could. Say for example the shield indicator ring around the ship. It could be used as a hit detector or an interface for directional shields (which should totally be a thing.) For now add the numbers, the labels, and fix the colors. The pretty stuff can come later

Red/hull = Health
Blue = Shields
Yellow = Power
Orange = Heat

**Comment by tsunamayo** [comment]
Yes I agree, I want to get more mileage into the combat system and then see what could make sense to really improve ergonomy. 
In the past I have spent too much time on some details to make it look good or like other game, that I found myself removing later on or that just get in the way. 
But it is okay to start thinking and discussing about it now.
